management-interface.go

```diff
@@ -104,6 +110,35 @@ func ParseNetworkConfig(filepath string) (*NetworkConfig, error) {
 	return config, nil
 }
 
+// WriteLocationData writes the location values to the location data file.
+// If it doesn't exist, it is created.
+func WriteLocationData(filepath string, location LocationData) error {
+	outBuf, err := yaml.Marshal(location)
+	if err != nil {
+		return err
+	}
+	return ioutil.WriteFile(filepath, outBuf, 0640)
+}
+
+// ParseLocationFile retrieves values from the location data file.
+func ParseLocationFile(filepath string) (*LocationData, error) {
+
+	// Create a default config
+	location := &LocationData{Latitude: 0.0, Longitude: 0.0}
```

Comment:
There's no need to set the default values. Go guarantees that values are initialised to their zero values.